File: apps/institution-api/Models/InstitutionLocationPublication.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class InstitutionLocationPublication
    {
        public int InstitutionLocationPublicationId { get; set; }
        public int? InstitutionLocationId { get; set; }
        public Guid LocationPublicationGuid { get; set; }
        public bool? IsPublishedOnWebsite { get; set; }
        public string LocationPublicationStatus { get; set; }
        public string SubRegion { get; set; }
        public string PublicationDetailStatus { get; set; }
        public DateTime? PublicationBusinessStartDate { get; set; }
        public DateTime? PublicationBusinessEndDate { get; set; }

        public virtual InstitutionLocation InstitutionLocation { get; set; }
    }
}

File: apps/planmp/src/libs/common/models/institution-location-publication.ts


import { InstitutionLocation } from './institution-location';

export class InstitutionLocationPublication {
    public institutionLocationPublicationId: number = 0;
    public institutionLocationId: number;
    public locationPublicationGuid: string = '';
    public isPublishedOnWebsite: boolean;
    public locationPublicationStatus: string = '';
    public subRegion: string = '';
    public publicationDetailStatus: string = '';
    public publicationBusinessStartDate: Date;
    public publicationBusinessEndDate: Date;
    public institutionLocation: InstitutionLocation = new InstitutionLocation();

    constructor(init? : Partial<InstitutionLocationPublication>) {
        
        Object.assign(this, init);
    }
}

